src/SFA.DAS.Funding.SystemAcceptanceTests/Helpers/WaitHelper.cs

```diff
@@ -2,7 +2,7 @@
 {
     public static async Task WaitForIt(Func<bool> lookForIt, string failText)
     {
-        var endTime = DateTime.Now.Add(TimeSpan.FromSeconds(240));
+        var endTime = DateTime.Now.Add(TimeSpan.FromSeconds(180));
```

Comment:
you could have these values as a config value if you wanted then we can easily change them when needed